src/yoti_common/attribute.converter.js

```diff
@@ -47,8 +60,25 @@ module.exports.AttributeConverter = class AttributeConverter {
         return new ImageJpeg(value);
       case CONTENT_TYPE_PNG:
         return new ImagePng(value);
+      case CONTENT_TYPE_MULTI_VALUE:
+        return AttributeConverter.convertMultiValue(value);
+      case CONTENT_TYPE_INT:
+        return parseInt(value.toUTF8(), 10);
```

Comment:
Could we add named parameters here? Just makes it clearer what the `10` is denoting 
```suggestion
        return parseInt({string: value.toUTF8(), radix: 10});
```

Reply:
I don't think we can do this neatly with JS. We could do one of the following:
- Assign the parameters to descriptive variables then pass those in
- Use a description constant such as `BASE_10` or `RADIX_10`
- Move `parseInt()` into a descriptive method, e.g. `parseIntFromBase10String()` that takes only the string.

It might be neater to leave this as is?

Reply:
Sure, leaving it is fine. I didn't realise the suggestion returned it as a JSON